Content.Server/Defusable/WireActions/ActivateBombWireAction.cs

```diff
@@ -0,0 +1,53 @@
+using Content.Server.Defusable.Components;
+using Content.Server.Defusable.Systems;
+using Content.Server.Doors.Systems;
+using Content.Server.Wires;
+using Content.Shared.Defusable;
+using Content.Shared.Doors;
+using Content.Shared.Doors.Components;
+using Content.Shared.Wires;
+
+namespace Content.Server.Defusable.WireActions;
+
+public sealed class ActivateBombWireAction : ComponentWireAction<DefusableComponent>
+{
+    public override Color Color { get; set; } = Color.Lime;
+    public override string Name { get; set; } = "wire-name-bomb-live";
+
+    public override StatusLightState? GetLightState(Wire wire, DefusableComponent comp)
+        => comp.BombLive ? StatusLightState.Off : StatusLightState.BlinkingFast;
+
+    public override object StatusKey { get; } = DefusableWireStatus.LiveIndicator;
+
+    public override bool Cut(EntityUid user, Wire wire, DefusableComponent comp)
+    {
+        if (comp.BombLive)
+        {
+            EntityManager.System<DefusableSystem>().DefuseBomb(wire.Owner, comp);
+        }
+        return true;
+    }
+
+    public override bool Mend(EntityUid user, Wire wire, DefusableComponent comp)
+    {
+        // bomb is defused why do you want it back on again
```

Comment:
shitsec leaves defused bomb in armoury so smart syndie rearms it and saves tc, should make rearming a thing
even the nuke has it

Reply:
i disagree, if security manages to defuse the bomb without being sent to hell, then that's great for them